### ISSUE TITLE
Users unable to log out from back end

### ISSUE THREAD

**Issue by EllisCrowther** [bueltge/adminimize#128]
Version Information
* WordPress: 5.3.2
* Multisite: false
* Adminimize: 1.11.6

Steps to Reproduce
1. Tried to make contributors admin more streamlined, got rid of a lot of options.
2. Updated
3. Lost the ability to log out from the back end, but not from the front end

What I Expected:

For contributors to just be able to log in/out, write/edit posts, add media, view comments, and edit their profile.

What Happened Instead:

All of the above, but they're unable to log out from the back end.

Screenshot or Video:

![adminimize](https://user-images.githubusercontent.com/45557306/76617249-d2019f00-651d-11ea-9d5e-5f7b9e892170.png)

I'm not sure if this is due to an incompatibility with the WPFront User Role Editor plugin, it's not a huge issue as contributors are still able to log out using the front end, but any support with this would be appreciated!

Thanks!

**Comment by bueltge** [comment]
I mean thats no issue on the plugin. Check that you have the role  'Adminstrator' with the capability 'manage_options' - thats the part there is necessary to get access to the settings page of Adminimize, not more. So also to remove the menu item you will get access via the URL, therefore do you not see that menu inside the backend.

**Comment by EllisCrowther** [comment]
Hi,

I have the 'administrator' role, which has the capability 'manage_options', and I am able to edit the settings from my account, as you can see in the screenshots above.

The issue is when I log in to a different account that has the 'contributor' role, where the login button is hidden, despite it not being hidden within these settings.

Would you be able to help with this issue?

**Comment by bueltge** [comment]
Maybe. Has you hide the login-area via Adminimize? 
![image](https://user-images.githubusercontent.com/133425/77070619-d67cfa80-69ea-11ea-8b45-bc3d6903bad6.png)
IF you hide them, the user has no option to logout inside the back-end. Please export your settings and upload here, so we should see is that option active.

**Comment by EllisCrowther** [comment]
Hi,

Thanks for your response, I haven't hidden the user info options, please see the screenshot below.

![Capture](https://user-images.githubusercontent.com/45557306/77247293-88a10600-6c27-11ea-8a18-f99b072720ad.PNG)

Is there any way that I can export all of my options so you can see them all? 

Thanks!

**Comment by bueltge** [comment]
Yes, the export option is also on the settings page, scroll down to see them.

**Comment by EllisCrowther** [comment]
Great, I found the option, please see attached below!

[mw_adminimize-settings-export-03-23-2020.zip](https://github.com/bueltge/adminimize/files/4368182/mw_adminimize-settings-export-03-23-2020.zip)
